qml/flightreservation.cpp

```diff
@@ -27,9 +27,14 @@
 FlightReservation::FlightReservation(QObject* parent): QObject(parent)
 {
     QDBusConnection dbus = QDBusConnection::sessionBus();
-    QDBusInterface* interface = new QDBusInterface("org.kde.kdenow", "/Flight");
+    m_interface = new QDBusInterface("org.kde.kdenow", "/Flight");
+    dbus.connect("org.kde.kdenow", "/Flight", "org.kde.kdenow.flight",
+                 "update", this, SLOT(onMapReceived()));
```

Comment:
I have intentionally, kept this as SLOT(onMapReceived()) because, making it &FlightReservation::onMapReceived (which is not of type consr char* ), fails to compile.